plugins/network-elements/nuage-vsp/src/com/cloud/network/guru/NuageVspGuestNetworkGuru.java

```diff
@@ -409,8 +472,8 @@ private HostVO getNuageVspHost(long physicalNetworkId) throws InsufficientVirtua
             nuageVspHost = _hostDao.findById(config.getHostId());
             _hostDao.loadDetails(nuageVspHost);
         } else {
-            throw new InsufficientVirtualNetworkCapacityException("Nuage VSD is not configured on physical network ", PhysicalNetwork.class, physicalNetworkId);
+            throw new CloudRuntimeException("Nuage VSD is not configured on physical network " + physicalNetworkId);
```

Comment:
Changed code to <code>throw new CloudRuntimeException("There is no Nuage VSP device configured on physical network " + physicalNetworkId)</code>. We can't identify about the Nuage VSP device that failed since there is none configured.